### DISCUSSION TITLE
JSOC Project Introduction

### DISCUSSION THREAD

**Discussion by Dsantra92** [JuliaImages/Images.jl#944] [JSoC]
Hi mentors :wave: ,
I am 2nd year undergrad. I am interested in developing the `Setreo Matching` package mentioned in the project list. I have gone through some of the relevant papers in this field (and trying to code them using Julia for now).

**Comment by zygmuntszpak** [comment]
Welcome, and thank you very much for expressing an interest in that particular project. Look forward to seeing how you progress with your initial implementation attempt. Which particular algorithm are you starting with?

**Comment by Dsantra92** [comment]
I thought it will be good to start with one of the papers mentioned in the projects page. So after reading a few, I started with the slanted support windows algorithm.
